python/__init__.py

```diff
@@ -106,6 +106,7 @@ class Panda(object):
   SAFETY_HONDA = 1
   SAFETY_TOYOTA = 2
   SAFETY_HONDA_BOSCH = 4
+  SAFETY_CHRYSLER = 8
```

Comment:
the python code is kind of deprecated, but for consistency I would set this to 9